content/en/boilerplates/revision-tags-usage.md

```diff
@@ -1,5 +1,5 @@
 ---
 ---
-Consider a cluster with two revisions installed, `1-9-5` and `1-10-0`. The cluster operator creates a revision tag `prod-stable`,
-pointed at the older, stable `1-9-5` version, and a revision tag `prod-canary` pointed at the newer `1-10-0` revision. That
+Consider a cluster with two revisions installed, `{{< istio_previous_version_revision >}}-5` and `{{< istio_full_version_revision >}}`. The cluster operator creates a revision tag `prod-stable`,
+pointed at the older, stable `{{< istio_previous_version_revision >}}-5` version, and a revision tag `prod-canary` pointed at the newer `{{< istio_full_version_revision >}}` revision. That
```

Comment:
```suggestion
Consider a cluster with two revisions installed, `{{< istio_previous_version_revision >}}-1` and `{{< istio_full_version_revision >}}`. The cluster operator creates a revision tag `prod-stable`,
pointed at the older, stable `{{< istio_previous_version_revision >}}-1` version, and a revision tag `prod-canary` pointed at the newer `{{< istio_full_version_revision >}}` revision. That
```

Reply:
addressed in ed722bf8e9211aa0ec9da36163ae8e3bd11ffb6d